examples/crypto-usage-tiingo-pandas.ipynb

```diff
@@ -0,0 +1,131 @@
+#%%
+#%% md
+
+# Tiingo Python for Cryptocurrencies
+
+
+This notebook shows basic usage of the `tiingo-python` library. If you're running this on `mybinder.org`, you can run this code without installing anything on your computer. You can find more information about what available at the [Tiingo website](https://api.tiingo.com/docs/general/overview), but this notebook will let you play around with real code samples in your browser.
+
+If you've never used `jupyter` before, I recommend this [tutorial](https://www.datacamp.com/community/tutorials/tutorial-jupyter-notebook) from Datacamp.
+
+#%% md
+
+## Basic Setup
+
+First, you'll need to provide your API key as a string in the cell below. If you forget to do this, the notebook cannot run. You can find your API key by visiting [this link](https://www.tiingo.com/account/api/token) and logging in to your Tiingo account.
+
+#%%
+
+!pip3 install os
+!pip3 install websocket
+!pip3 install json
+!pip3 install Tiingo.exceptions
+!pip3 install matplotlib
+!pip3 install pandas
+!pip3 install numpy
+!pip3 install pandas-datareader
+
+#%%
+
+TIINGO_API_KEY = '8c35e8407d4da9c94e39bdf0dbc7913a899c2377'
```

Comment:
I think you may want to invalidate this API key so other people can't reuse it / rewrite git history to not include this directly in your notebook.

Reply:
thanks hydrosquall, token is reset. Can you recommend a good tutorial on setting up passwords in environments if we are developing on one machine and testing on another? Keychain perhaps ?

Reply:
In the python ecosystem, we use

- https://github.com/motdotla/dotenv

I haven't used this personally, but the world of Python seems to have this as a similar project

- https://github.com/theskumar/python-dotenv

Most CI providers (Jenkins, Gitlab, CircleCI etc) will offer a way to supply environment variables via a UI specific to their platform. Keychain is a good option if you're using Mac everywhere.